### PR TITLE
automatically increment the image gallery version.

### PR DESCRIPTION
**Pull request by SalimKainos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-14721

### Change description ###

Update image gallery version automatically 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by thomast1906** [comment]
Have pinged on Slack - curious about this

File: azure-pipelines.yml [hunk 1]
```diff
@@ -21,5 +40,8 @@ jobs:
         displayName: 'Build'
         condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/heads/master'))
         inputs:
+          templatePath: $(templatePath)
           command: build
           azureSubscription: dts-management-prod-intsvc
+          variables: |
+              azure_image_version=$(get_azure_image_version.azure_image_version)
```
Comment (thomast1906) [inline azure-pipelines.yml:47]: I dont think this is simplified - lets catchup

Comment (cpareek) [reply azure-pipelines.yml:47]: Morning Thomas - sure - available when you are..

Comment (thomast1906) [reply azure-pipelines.yml:47]: need that morning coffee, it looks fine on second look

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**